pymobiledevice3/cli/webinspector.py

```diff
@@ -295,3 +210,109 @@ def cdp(lockdown: LockdownClient, host, port):
     udid = lockdown.udid
     uvicorn.run('pymobiledevice3.cli.webinspector:create_app', host=host, port=port, factory=True,
                 ws_ping_timeout=None, ws='wsproto', loop='asyncio')
+
+
+class JsShell(ABC):
+
+    def __init__(self, lockdown: LockdownClient, timeout: float, app: str):
```

Comment:
Might be easier the add staticmethod / classmethod that initialize (and selects the page)